File: database/db_connector.js
//npm install mysql

// This file connects to the remote Google Cloud Platform
// It is currently not running security features in the interest of being fully operational for integration on Saturday
// At the moment, the database is allowing accesses from all networks to prevent problems with authentication delaying progress on other code aspects
class db_connector{
  constructor() {
    console.log('Beginning');

    var mysql = require('mysql');

    this.con = mysql.createConnection({
      host: "35.184.92.72",
      user: "Group9",
      password: "food@4193",
      database: "production"
    });

    this.con.connect(function(err) {
      if (err) throw err;
      console.log("Connected");
    });
  }

  add_user(username, hashed_password) {
      console.log("Posting")
      db.con.query("INSERT INTO users (username, hashed_password) VALUES (\x22" + username + "\x22, \x22" + hashed_password + "\x22);", function (err, result) {
        if (err) throw err;
        console.log("Result: " + result);
      });
  }
}

File: database/src/db_connector.java
//install mysql-connector-j

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.sql.Statement;

public class db_connector
{
    public db_connector()
    {
        System.out.println("CREATING DATABASE");

        /*String url = "jdbc:mysql://35.184.92.72/production";
        String username = "Group9";
        String password = "food@4193";
    
        System.out.println("Attempting to Connect");
        //Class.forName("com.mysql.jdbc.Driver");
        try (Connection con = DriverManager.getConnection(url, username, password)) {
            System.out.println("Connected");
            String query = "INSERT INTO users (username, hashed_password) VALUES (\"TEST1\", \"PASSWORD123\");" ;
            PreparedStatement state = con.prepareStatement(query);
            state.execute(query);
            //result.close();
            } catch (SQLException e) {
            throw new IllegalStateException("Cannot connect the database!", e);
        }*/
    }

}